cmd/tk/workflow.go

```diff
@@ -25,6 +25,11 @@ func workflowFlags(fs *pflag.FlagSet) *workflowFlagVars {
 	return &v
 }
 
+func applyDeleteFlags(fs *pflag.FlagSet, opts *kubernetes.ApplyDeleteOpts) {
```

Comment:
No need to pass in the `opts`, rather create them in here and return a pointer to the struct